peps/pep-0729.rst

```diff
@@ -0,0 +1,413 @@
+-------------------------------------------------------------------
+
+The SC has the entire language in its remit, and is not well-placed to make
+decisions that are purely about the type system -- if only because they don't have
+the time to deal with type system arcana alongside their other responsibilities.
+This is similar in spirit to the reasons why the Steering Council sometimes uses
+PEP delegation.
+
+Specification
+=============
+
+We propose the creation of a new group, the Typing Council. This group will
+be responsible for developing and maintaining the Python type system, and
+for solving the above problems.
+
+The "operations and process" section describes how this group would operate and
+be governed.
+
+The more exciting "projects" section describes solutions to the above problems
+that the Typing Council could shepherd.
+
+Mandate
+-------
+
+The Typing Council's mandate is to ensure that the Python type system is:
+
+* **Useful**: The type system should serve common use cases. As identified
+  by :pep:`484`, the primary use case is static analysis, but there are others,
+  such as runtime type checking, static compilation, IDE support, and documentation.
+  The Typing Council should consider all of these use cases when making decisions,
+  and be open to supporting additional use cases as they come up.
+* **Usable**: The type system should be easy to use for Python developers. It
+  should be ergonomic to write well-typed Python code that is accepted by type
+  checkers. There should be good documentation for the type system.
+* **Stable**: As the type system matures, users should be able to rely on their
+  typed code continuing to work and be able to trust their mental model for the
+  type system. Changes should be made with care and in a way
+  that minimizes disruption. Nevertheless, the type system should be able to
+  evolve, and type checkers need not be held to the same strict compatibility
+  standards as the Python language itself.
+
+Operations and process
+----------------------
+
+The council would have three members, comprised of prominent community members,
+such as Python core developers and maintainers of major type checkers. The members
+should include people affiliated with a variety of projects related to type checking,
+which may include type checkers, CPython, typeshed, or other projects.
+
+The Steering Council would appoint the initial Typing Council. There is no term
+limit for council members. Council members may resign their position at any time.
+There is a norm that at least one person will voluntarily resign from the
+Typing Council each year.
+
+To determine replacements, nominations will be collected from the typing
+community. Self-nominations are allowed. The Typing Council will then decide
+the replacement member(s) from the nominees. The expectation is that this would
+be done by fiat, but the Typing Council can choose a replacement by any means
+they see fit, including a vote.
```

Comment:
Do the resigning members participate in selecting the replacements?

Reply:
My intent was that they could, three is a number that helps break ties